packages/dx-react-chart-material-ui/src/templates/legend/item.jsx

```diff
@@ -7,6 +7,8 @@ import ListItem from '@material-ui/core/ListItem';
 const styles = () => ({
   root: {
     alignItems: 'center',
+    paddingTop: 4,
+    paddingBottom: 4,
```

Comment:
In the Material Design theme, we often write `theme.spacing.unit` instead of numbers. See [example](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid-material-ui/src/templates/table-cell.jsx#L10).